MapboxMobileEvents.xcodeproj/project.pbxproj

```diff
@@ -1445,6 +1581,7 @@
 		404807CE1EA186D8008A6D50 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				APPLICATION_EXTENSION_API_ONLY = YES;
```

Comment:
The majority of this project file change have to do with the addition of a Today Widget.

This particular change though is what catches on build time if we ever use apis that don't work in extensions. Which will prevent this from happening in the future.